cmd/loadtest/main.go

```diff
@@ -43,6 +56,8 @@ var (
 
 // Run is the main function of this application. It returns a status exit code for main().
 func Run() int {
```

Comment:
i think that the best way to design this entire app is to do the following:
1. have a helper applicatino to pre-generate all transaction XDRs. similiar to the other app which creates acconuts beforehand.
1. load testing up will just load as much as necessary and submit them when the time comes

this will allow us to cache the premade txs instead of generating them as the test goes on (which takes time).

Reply:
I am considering writing a new app in Swift.  The only people who actually understand go-lang are too busy to maintain this one, and the changes we need are outgrowing my limited ability.